apps/docs/src/app/platform/component-docs/platform-forms/platform-datetime-picker/platform-datetime-picker-examples/platform-datetime-picker-basic-example.component.ts

```diff
@@ -0,0 +1,38 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { FormGroup, ValidatorFn, Validators } from '@angular/forms';
+import { FdDate, FdDatetime } from '@fundamental-ngx/core';
+
+@Component({
+    selector: 'fdp-datetime-picker-basic-example',
+    templateUrl: './platform-datetime-picker-basic-example.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PlatformDatetimePickerBasicExampleComponent {
+    storedDate: FdDatetime = FdDatetime.getToday();
+
+    datetimePickerForm = new FormGroup({});
+
+    requiredDateValidator: ValidatorFn[];
```

Comment:
`requiredDateValidator: ValidatorFn[] = [Validators.required]`

Reply:
missed it 🤦 thanks for noticing! fixed now